code/edit.html

```diff
@@ -0,0 +1,64 @@
+<html>
+<body>
+ 
+<table>
+    <tr><td style="font-size:30px"><b>Edit Your Code:</b></td></tr>
+    <tr>
+        <td colspan="3">
+            <textarea id="inputTextToSave" cols="150" rows="35" style="border-width:20px"></textarea>
+        </td>
+    </tr>
+    <tr>
+        <td>Filename to Save As:</td>
+        <td><input id="inputFileNameToSaveAs"></input></td>
+        <td><button onclick="saveTextAsFile()">Save Text to File</button></td>
+    </tr>
+    <tr>
+        <td>Select a File to Load:</td>
+        <td><input type="file" id="fileToLoad"></td>
+        <td><button onclick="loadFileAsText()">Load Selected File</button><td>
```

Comment:
`onclick` does not work in extensions. You can trigger the `loadFileAsText` function by a `js` code listening for a click event on the button.